Makefile

```diff
@@ -9,27 +9,21 @@ ifeq (, $(shell which richgo))
 $(warning "could not find richgo in $(PATH), run: go install github.com/kyoh86/richgo@latest")
 endif
 
-.PHONY: fmt lint test install_deps clean
+.PHONY: lint test clean
 
 default: all
 
-all: fmt test
+all: test
 
-fmt:
-	$(info ******************** checking formatting ********************)
-	@test -z $(shell gofmt -l $(SRC)) || (gofmt -d $(SRC); exit 1)
```

Comment:
So we no longer need `gofmt` because our linter handles it? We should change this to be a call out to the `golangci-lint` commandline utility then.

Reply:
See the 'lint' target in the same Makefile.

Reply:
Ah gotcha - but is `golangci-lint` going to format poorly formatted code?

Reply:
No. It will check if the format is correct. That's what the current `fmt` Makefile target does:

https://github.com/spf13/cobra/blob/a281c8b47b315fe428cf31ad5c38b526dc136f55/Makefile#L19-L20

This PR does not introduce any change in that regard.